forms/EmailField.php

```diff
@@ -1,54 +1,68 @@
 <?php
+
 /**
- * Text input field with validation for correct email format
- * according to RFC 2822.
- * 
+ * Text input field with validation for correct email format according to RFC 2822.
+ *
  * @package forms
  * @subpackage fields-formattedinput
  */
 class EmailField extends TextField {
-
+	/**
+	 * @return string
```

Comment:
This phpdoc is already on FormField::Type(), but with more documentation. You probably don't need to duplicate the whole block all the way down the inheritance chain; PHPDoc is by nature already inherited.